src/components/CostEstimationContent/CostOperation.tsx

```diff
@@ -0,0 +1,187 @@
+import React, { FunctionComponent, useContext, useState, useEffect } from "react";
+import { OverlayContext } from "@govtechsg/tradetrust-ui-components";
+import { useFetchGasCost } from "../../common/hooks/useFetchGasPrice";
+import { CostData } from "./types";
+import { CostModal } from "./CostModal";
+import { currentDateStr } from "../../utils";
+
+export const CostOperation: FunctionComponent = () => {
+  const [dateTime, setDateTime] = useState(currentDateStr());
+  const { price, gwei } = useFetchGasCost("ethereum", 30000);
+  const { price: maticPrice, gwei: maticGwei } = useFetchGasCost("polygon", 30000);
+  const priceFactor = gwei * 0.000000001 * price;
+  const maticPriceFactor = maticGwei * 0.000000001 * maticPrice;
+
+  const transferHoldershipGas = 43634;
+  const transferOwnershipGas = 324688;
+  const issueDocGas = 239523;
+  const burnDocGas = 56532;
+  const surrenderDocGas = 93435;
```

Comment:
why hardcode gas values?

Reply:
they are the estimated averaged gas for a number of runs for each transactions.

Reply:
at least put them outside of the component; static data should always live outside the component